examples/tensorflow-privacy/README.md

```diff
@@ -1,18 +1,20 @@
 ---
-tags: [basic, vision, fds, privacy, dp]
+tags: [DP, DP-SGD, basic, vision, fds, privacy]
 dataset: [MNIST]
 framework: [tensorflow]
 ---
 
 # Training with Sample-Level Differential Privacy using TensorFlow-Privacy Engine
 
-In this example, we demonstrate how to train a model with sample-level differential privacy (DP) using Flower. We employ TensorFlow and integrate the tensorflow-privacy Engine to achieve sample-level differential privacy. This setup ensures robust privacy guarantees during the client training phase.
+In this example, we demonstrate how to train a model with sample-level differential privacy (DP) using Flower. We employ TensorFlow and integrate the tensorflow-privacy engine to achieve sample-level differential privacy. This setup ensures robust privacy guarantees during the client training phase.
 
 For more information about DP in Flower please refer to the [tutorial](https://flower.ai/docs/framework/how-to-use-differential-privacy.html). For additional information about tensorflow-privacy, visit the official [website](https://www.tensorflow.org/responsible_ai/privacy/guide).
 
-## Environments Setup
+## Set up the project
 
-Start by cloning the example. We prepared a single-line command that you can copy into your shell which will checkout the example for you:
+### Clone the project
+
+Start by cloning the example project:
```

Comment:
Could we show the command below as in the other examples (multi line): https://github.com/adap/flower/tree/main/examples/quickstart-pytorch